engines/griffon/cutscenes.cpp

```diff
@@ -233,15 +262,21 @@ void GriffonEngine::intro() {
 			cnt = 0;
 			y--;
 		}
-
-		for (int i = 0; i <= 37; i++) {
+		
```

Comment:
You have a line with a trailing tab here. You may want to check the settings in your text editor as it can help you avoid those (for example Xcode has a setting `Automatically trim trailing whitespace` and a separate child option `Including whitespace-only lines` and I would recommend to toggle both on.